roxie/ccd/ccdmain.cpp

```diff
@@ -1326,7 +1326,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                     unsigned numThreads = roxieFarm.getPropInt("@numThreads", 0);
                     if (!numThreads)
                         numThreads = numServerThreads;
-                    unsigned port = roxieFarm.getPropInt("@port", ROXIE_SERVER_PORT);
```

Comment:
This breaks bare-metal...

Reply:
are other places ok to default in bare metal (e.g. line 1205) ?

Perhaps instead of conditional code here, check @port in bare-metal of farmers upfront, e.g. in block around line 656

Reply:
Others are inside #ifdef's that make them containerized only.